wolfcrypt/src/wc_port.c

```diff
@@ -2270,7 +2270,7 @@ int wolfSSL_CryptHwMutexUnLock(void)
 
     int wc_LockMutex(wolfSSL_Mutex* m)
     {
-        OS_MUTEX_Lock((OS_MUTEX*) m);
+        OS_MUTEX_LockBlocked((OS_MUTEX*) m);
```

Comment:
Is there a return code? Is this blocked version support in an embos?

Reply:
The blocked version is a standard part of the embOS API. I mixed the two up in my original PR as I was using a FreeRTOS -> embOS conversion guide originally.

There is a return code, but I believe it is meaningless in our context:

**Return value**
The counter value of the mutex.
A value greater than one denotes the mutex was already locked by the calling task.


In the non-blocking version a zero return means that locking failed.